src/libsync/syncfilestatustracker.cpp

```diff
@@ -168,9 +169,7 @@ void SyncFileStatusTracker::decSyncCount(const QString &relativePath, EmitStatus
 
 void SyncFileStatusTracker::slotAboutToPropagate(SyncFileItemVector& items)
 {
-    // New sync, clear the sync counts to reduce the impact of a bug that would cause unsymetrical inc/dec
     Q_ASSERT(_syncCount.isEmpty());
-    _syncCount.clear();
```

Comment:
I don't think anything can increment this between slotSyncFinished and here. I was even considering removing the assert.